content/realtime/channel-lifecycle.textile

```diff
@@ -0,0 +1,141 @@
+---
+title: Channel Metadata and Lifecycle Events
+section: realtime
+index: 15
+jump_to:
+  Help with:
+    - Overview#overview
+    - Metachannels#metachannels
+    - Lifecycle Events#lifecycle-events
+    - Reactor Rules#reactor
+    - Permissions#permissions
+  Channel API Types:
+    - ChannelDetails#channel-details
+    - ChannelStatus#channel-status
+    - Occupancy#occupancy
+---
+
+h2(#overview). Overview
+
+The realtime "channel lifecycle":#lifecycle-events and "occupancy":#occupancy events are published onto "metachannels":#metachannels that can be subscribed to using our existing "realtime libraries":/realtime. This is often used in conjunction with the "Ably Reactor":/reactor, which allows you to send this data to your "serverless endpoints":/general/functions, "queues":general/queues, and "streams":general/firehose.
+
+The "REST API":/rest-api can also be used "enumerate all active channels":/rest/channel-lifecycle#enumeration-rest, or obtain the status of an "individual channel":#metadata-rest. At present, REST "requests":/rest/usage#request for lifecycle events will not count towards your message limit. If you are making use of "Reactor":/reactor however, then each message sent over Reactor *will* count towards your message limits.
+
+h2(#use-cases). Use cases
+
+Having access to channel metadata can provide numerous benefits. In a scenario where the number of subscribers to a channel goes well beyond a hundred, usage of other options such as "presence":/core-features/presence become less effective. You could instead make use of our "occupancy events":#occupancy and "channel metadata":#metachannels to check the number of active subscribers, getting accurate values easily.
+
+Equally, you may want to only publish data when you know someone is subscribed for your data. Therefore, "channel lifecycle events":#lifecycle-events can notify you when a channel becomes active, and can notify you when the last subscriber leaves the channel.
+
+If you need to be able to query channel metadata at any point, you can make use of the "channel metadata REST APIs":/rest/channel-lifecycle#metadata-rest to inspect the state of individual channels, or "enumerate":/rest/channel-lifecycle#enumeration-rest all active channels in the system.
+
+h2(#tutorials). Tutorials
+
+Check out some of our "tutorials":/tutorials to see examples of using metachannels and channel lifecycles.
+
+Through a realtime connection you can subscribe to both "channel lifecycle":#lifecycle-events and "occupancy":#occupancy events. This is done through a channel's "metachannel":#metachannels.
```

Comment:
'Channel's metachannel' doesn't really make sense. 

-> This is done using the metachannel as explained next.